src/Form/_index.scss

```diff
@@ -492,7 +492,7 @@ select.form-control {
 }
 
 .pgn__form-radio-input {
-  border-radius: var(--pgn-size-form-control-border-radio-indicator-radius);
+  border-radius: var(--pgn-size-form-control-border-radio-indicator-radius) !important;
```

Comment:
Let's look at the approach of increasing specificity instead of adding `!important`

```
.pgn__form-radio .pgn__form-radio-input
```

Reply:
I changed it :), that fits better, thank you :)